lib/resque.rb

```diff
@@ -187,6 +187,46 @@ def enqueue_front
     end
   end
 
+  attr_writer :per_worker_stats
+  def per_worker_stats
+    if defined? @per_worker_stats
+      @per_worker_stats
+    else
+      @per_worker_stats = true
+    end
+  end
```

Comment:
FEATURE: `per_worker_stats`

Could be PRed to master, it's a feature flag.

Reply:
Definitely not at Github - the code to collect these stats is literally *deleted* from `github/resque`.

In other environments, it's pretty much equally ergonomic (IMO) to do `Resque.per_worker_stats = ENV['BLAH']` in an application's Resque config.